pex/pex.py

```diff
@@ -490,6 +494,10 @@ def cmdline(self, args=()):
     cmds.extend(args)
     return cmds
 
+  def _merge_split(self, *paths):
+    merged_paths = ":".join(filter(None, paths))
+    return filter(None, merged_paths.split(':'))
+
```

Comment:
could reduce this to a one liner list comprehension:

```
def _merge_split(self, *paths):
  return [p for p in ':'.join(paths).split(':') if p]
```

with the `if p` bit doing the same as the `filter(None, ...)` above.